### DISCUSSION TITLE
Allow to style "Before" and "After" fields in the Post Info widget when using Dynamic Tags

### DISCUSSION THREAD

**Discussion by Makramgit** [elementor/elementor#15475] [Feature Request]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
In the Post Info Widget, when using a custom dynamic field (ACF or Other) the "Before" prefix that you input in the advanced pop up is not assigned a style when rendered on the front end. When using a regular predefined filed (Author, date, time...) the before prefix is properly assigned a span class "elementor-post-info__item-prefix"

**Describe the solution you'd like**
Assign a css span class to prefix (and suffix) of dynamic fields so it becomes possible to style them differently. Ideally also, the placement of the input fields in the editor can remain on the same panel just like when using a regular fields.

**Comment by Makramgit** [comment]
Is there any workaround to style the prefix for ACF fields?

**Comment by yeldarb420** [comment]
I have been able to target the prefix of terms with `selector .elementor-post-info__item-prefix`, but not custom items (_supplied via a toolset field, one text, one date_). 
Terms they seem to be putting a span around the prefix, but not custom (have not checked on the treatment of the other options). 
Besides the normal styling, the list of terms can be targeted with `selector .elementor-post-info__terms-list a`
The entire custom item, prefix & all, can be targeted with `.elementor-icon-list-text`, but unfortunately just one giant span so not much else can be done beyond the normal. 
Makes no sense that they would on one but not the others. So I have half my items perfect & the others I can't target the prefix or the rest besides the normal controls.

I have found "_post-info.php_" in "_wp-content/plugins/elementor-pro/modules/theme-elements/widgets_". I searched for all three classes & found where the items are constructed. But not sure exactly what to swap out? Currently gotta move onto more pressing issues for now, but would love to get this resolved sooner then later, it's simple, but my "skills" stop there. Will post back if I figure out anymore. 

One other related issue, when the "custom" date field is empty, I am getting `<br />
<b>Warning</b>: date() expects parameter 2 to be integer, string given in <b>/home/photoncr/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/toolset/tags/toolset-date.php</b> on line <b>61</b><br />`
The "custom" text field & "terms" don't appear if empty, exactly what I want.
